### PR TITLE
py-ttfquery: bump version 2.0.0b1

### PR DESCRIPTION
**Pull request by catap**
###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on
macOS 11.5.2 20G95 x86_64
Xcode 12.5.1 12E507

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?

### CODE REVIEW

**Review by reneeotten — CHANGES_REQUESTED** [review]
a few comments/suggestions below. 

Also, we very much appreciate (new) contributors and welcome PRs for updates and new ports. So please keep them going! However, you might want to consider pacing them a little bit so that people actually have a chance to review/merge them 😉. There aren't all that many people here that keep the PRs moving and many (most?) are working on their "own" MacPorts things as well beside having a day job 😉

File: python/py-ttfquery/Portfile
Comment (catap) [reply python/py-ttfquery/Portfile:8]: Oh, this one is quite wired to be honest. Let me try to make things straight.

First at all it is very old library: https://pypi.org/project/TTFQuery/#history and you may see that the last stable release was in 2012, and the beta release which I've used in 2018.

This beta release is the last released yet, and the first one which supports python3.

Historically this project was hosted at [sourceforge](https://sourceforge.net/p/ttfquery/code/) by someone with name @mcfletch and the same account name is used to host the newest version at [GitHub](https://github.com/mcfletch/ttfquery/commits/master).

As you may see the date of release the last beta version at pypi matches the last commit.

So, I'm not sure that it is a beta version, probably it is the last released version and the only one which can be used at modern python :)

File: python/py-ttfquery/Portfile
Comment (catap) [reply python/py-ttfquery/Portfile:30]: Yes, it does. And differences between this two version isn't something huge that sems as broken compatibility: https://github.com/mcfletch/ttfquery/compare/76d6b...6988e5

File: python/py-ttfquery/Portfile
Comment (catap) [reply python/py-ttfquery/Portfile:7]: Thanks, it defently makes port simpler.

**Comment by catap** [comment]
> However, you might want to consider pacing them a little bit so that people actually have a chance to review/merge them 😉. There aren't all that many people here that keep the PRs moving and many (most?) are working on their "own" MacPorts things as well beside having a day job 😉

I do understand it and can wait until it is merged.

I'm almost finished with achiving my goal to install [beets](https://github.com/macports/macports-ports/pull/12132) from macports, and I can live for a while with installed ports from branches when review is going.

When I've started it I expect that I'll open a few PR not more than 30. Anyway, the end game almsot here => I've decided to make it right and enjoy the ride after that.

File: python/py-ttfquery/Portfile
Comment (reneeotten) [reply python/py-ttfquery/Portfile:8]: okay, thanks for the explanation - then it makes sense to update to this version.

File: python/py-ttfquery/Portfile
Comment (reneeotten) [inline python/py-ttfquery/Portfile:30]: with the recent changes to the `fonttools` port, upon your request, you should now likely depend on `port:py${python.version}-fonttools`

Comment (reneeotten) [reply python/py-ttfquery/Portfile:30]: upstream also lists `numpy` as a dependency in their `setup.py`: `install_requires = ['fonttools','numpy'],`

Also, I would assume that numpy and fonttools are not only needed at build time but also at runtime, correct? In that case they should be listed as `depends_lib`.

Comment (reneeotten) [reply python/py-ttfquery/Portfile:30]: the port also installs `console_scripts` and if you check, for example, `/opt/local/bin/ttffamily-3.9` you'll see that it uses `importlib.metadata`. That is part of Python since 3.8+ but for 2.7 and 3.8 you should add a dependency on the `py-importlib-metadata` port.

Comment (catap) [reply python/py-ttfquery/Portfile:30]: @reneeotten thanks, tooo many ports and I've missed. I thought it was needed only at beet's subports.

About dependencies: I've moved to `depends_lib` as you suggested. Anyway,  the script works without `py-importlib-metadata`
```
√ macports-ports % sudo port uninstall py37-importlib-metadata                                                                      (git)-[-py-ttfquery]
√ macports-ports % /opt/local/bin/ttffamily-3.7                                                                                     (git)-[-py-ttfquery]
INFO:ttfquery.ttffiles:Registry already populated
INFO:ttfquery.ttffiles:166 fonts available
F github-octicons
  /Users/catap/Library/Fonts/octicons.ttf
F Weather Icons
  /Users/catap/Library/Fonts/weathericons.ttf
F all-the-icons
  /Users/catap/Library/Fonts/all-the-icons.ttf
F FontAwesome
  /Users/catap/Library/Fonts/fontawesome.ttf
F file-icons
  /Users/catap/Library/Fonts/file-icons.ttf
...
````
and when I've installed it:
```
√ macports-ports % sudo port install --unrequested py37-importlib-metadata                                                          (git)-[-py-ttfquery]
--->  Computing dependencies for py37-importlib-metadata
The following dependencies will be installed: 
 py37-typing_extensions
 py37-zipp
Continue? [Y/n]: y
--->  Fetching archive for py37-typing_extensions
--->  Attempting to fetch py37-typing_extensions-3.10.0.2_0.darwin_20.x86_64.tbz2 from https://cph.dk.packages.macports.org/py37-typing_extensions
--->  Attempting to fetch py37-typing_extensions-3.10.0.2_0.darwin_20.x86_64.tbz2.rmd160 from https://cph.dk.packages.macports.org/py37-typing_extensions
--->  Installing py37-typing_extensions @3.10.0.2_0
--->  Activating py37-typing_extensions @3.10.0.2_0
--->  Cleaning py37-typing_extensions
--->  Fetching archive for py37-zipp
--->  Attempting to fetch py37-zipp-3.5.0_0.darwin_20.x86_64.tbz2 from https://cph.dk.packages.macports.org/py37-zipp
--->  Attempting to fetch py37-zipp-3.5.0_0.darwin_20.x86_64.tbz2.rmd160 from https://cph.dk.packages.macports.org/py37-zipp
--->  Installing py37-zipp @3.5.0_0
--->  Activating py37-zipp @3.5.0_0
--->  Cleaning py37-zipp
--->  Fetching archive for py37-importlib-metadata
--->  Attempting to fetch py37-importlib-metadata-4.6.4_0.darwin_20.noarch.tbz2 from https://cph.dk.packages.macports.org/py37-importlib-metadata
--->  Attempting to fetch py37-importlib-metadata-4.6.4_0.darwin_20.noarch.tbz2.rmd160 from https://cph.dk.packages.macports.org/py37-importlib-metadata
--->  Installing py37-importlib-metadata @4.6.4_0
--->  Activating py37-importlib-metadata @4.6.4_0
--->  Cleaning py37-importlib-metadata
--->  Scanning binaries for linking errors
--->  No broken files found.                             
--->  No broken ports found.
√ macports-ports % /opt/local/bin/ttffamily-3.7                                                                                     (git)-[-py-ttfquery]
INFO:ttfquery.ttffiles:Registry already populated
INFO:ttfquery.ttffiles:166 fonts available
F github-octicons
  /Users/catap/Library/Fonts/octicons.ttf
F Weather Icons
  /Users/catap/Library/Fonts/weathericons.ttf
F all-the-icons
  /Users/catap/Library/Fonts/all-the-icons.ttf
F FontAwesome
  /Users/catap/Library/Fonts/fontawesome.ttf
F file-icons
  /Users/catap/Library/Fonts/file-icons.ttf
F Material Icons
  /Users/catap/Library/Fonts/material-design-icons.ttf
F Symbol
  /System/Library/Fonts/Symbol.ttf
F .SF NS Mono
  /System/Library/Fonts/SFNSMono.ttf
  /System/Library/Fonts/SFNSMonoItalic.ttf
F Apple Braille

```

Comment (catap) [reply python/py-ttfquery/Portfile]: @reneeotten you're right. Thanks!

**Review by reneeotten — APPROVED** [review]

**Pull request merged**
